css/paragraphs.css

```diff
@@ -163,3 +163,7 @@ html.dialog-open {
   background: transparent url(../images/unpublish.svg) 6px center no-repeat;
   background-size: 14px;
 }
+
+.paragraphs-type-clone-readonly-field {
+  background-color: #f1f1f1 !important;
+}
```

Comment:
Let's avoid using `!important` if we can, perhaps with greater specificity like `.paragraphs-admin-bundle-clone-create-form .paragraphs-type-clone-readonly-field` (I tested with `input.paragraphs-type-clone-readonly-field` but that leaves out the text area so isn't quite enough)

Reply:
Done. You might want to change this in 'Content Type Clone' 😉

Reply:
Fair enough. ;) 
https://github.com/backdrop-contrib/content_type_clone/issues/7